packages/upload/doc/basic.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <el-upload
+    class="upload-demo"
+    action="https://jsonplaceholder.typicode.com/posts/"
+    :on-preview="handlePreview"
+    :on-remove="handleRemove"
+    :before-remove="beforeRemove"
+    :multiple="multiple"
+    :limit="3"
+    :on-exceed="handleExceed"
+    :file-list="fileList"
+  >
+    <el-button size="small" type="primary">Click to upload</el-button>
+    <template #tip>
+      <div class="el-upload__tip">jpg/png files with a size less than 500kb</div>
+    </template>
+  </el-upload>
+</template>
+<script lang="ts">
+export default {
```

Comment:
Should we use a setup function or <script lang="ts" setup> ?

Reply:
Nope, setup script is still experimental and cannot properly handle props.